### ISSUE TITLE
TypeError: Cannot read property 'useMemo' of null

### ISSUE THREAD

**Issue by MakiychukYaroslav** [troberts-28/react-native-timer-picker#15]
```
import { TimerPicker } from 'react-native-timer-picker';
...
  <TimerPicker
          padWithNItems={2}
          hourLabel=":"
          minuteLabel=":"
          secondLabel=""
          styles={{
            theme: 'dark',
            backgroundColor: '#202020',
            pickerItem: {
              fontSize: 34,
            },
            pickerLabel: {
              fontSize: 32,
              marginTop: 0,
            },
            pickerContainer: {
              marginRight: 6,
            },
          }}
        />
```
packages:
 "react": "17.0.2",
  "react-native": "0.67.3",
"react-native-timer-picker": "^1.2.7",

<img width="405" alt="image" src="https://github.com/troberts-28/react-native-timer-picker/assets/23318865/6e011946-12ed-46a9-bbee-1757df3b6de4">

**Comment by MakiychukYaroslav** [comment]
An error occurred while trying to add a component to the project @troberts-28 
Could you please help with this?

**Comment by troberts-28** [comment]
Hi @MakiychukYaroslav, I'll take a look. Are you using Expo or raw React Native?

**Comment by MakiychukYaroslav** [comment]
Bare React-Native

**Comment by troberts-28** [comment]
Hi @MakiychukYaroslav!

This should be fixed for you in the latest version (v1.2.10) 🙌
